proto/regen/ecocredit/marketplace/v1/tx.proto

```diff
@@ -22,8 +22,8 @@ service Msg {
   // CancelSellOrder cancels a sell order and returns the funds from escrow.
   rpc CancelSellOrder(MsgCancelSellOrder) returns (MsgCancelSellOrderResponse);
 
-  // Buy creates credit buy orders.
-  rpc Buy(MsgBuy) returns (MsgBuyResponse);
```

Comment:
`MsgBuy`, etc. should also get deleted

Reply:
okay removed everything related to MsgBuy